client/core/trade.go

```diff
@@ -540,17 +548,22 @@ func (t *trackedTrade) tick() (assetCounter, error) {
 	for _, match := range t.matches {
 		switch {
 		case t.isSwappable(match):
+			log.Debugf("swappable match %v", match.id)
 			swaps = append(swaps, match)
 			sent += match.Match.Quantity
 			quoteSent += calc.BaseToQuote(match.Match.Rate, match.Match.Quantity)
 
 		case t.isRedeemable(match):
+			log.Debugf("redeemable match %v", match.id)
 			redeems = append(redeems, match)
 			received += match.Match.Quantity
 			quoteReceived += calc.BaseToQuote(match.Match.Rate, match.Match.Quantity)
 
 		case t.isRefundable(match):
+			log.Debugf("refundable match %v", match.id)
 			refunds = append(refunds, match)
+		default:
+			log.Debugf("match not ready for action: %v", match.id)
```

Comment:
Maybe this doesn't need to log if `status = MatchComplete`?

Reply:
They should be removed, but that's out of scope for this PR.